libfwupd/fwupd-client.c

```diff
@@ -305,19 +409,21 @@ fwupd_client_get_main_context (FwupdClient *self)
 /**
  * fwupd_client_set_main_context:
  * @self: A #FwupdClient
- * @main_ctx: #GMainContext
+ * @main_ctx: (nullable): #GMainContext or %NULL
```

Comment:
Would be good to clarify what `NULL` means.

Reply:
Fixed, ta.